src/relax/transform/utils.h

```diff
@@ -116,6 +134,123 @@ IRModule MakeGroupedFunctions(
     const std::unordered_map<const Object*, relay::GraphPartitioner::Group*>& partition,
     bool lift_constants = true);
 
+/*!
+ * \brief Check if the given StructInfo is a scalar tensor. The sinfo should be an instance of
+ * TensorStructInfo; its shape must be ShapeExpr.
+ * \param sinfo The StructInfo to be checked.
+ * \return true if the given StructInfo is a scalar tensor.
+ */
+bool IsScalarTensor(const StructInfo& sinfo);
```

Comment:
Looks like some functions are not used and no implemtation in cc files. Please remove them

Reply:
OK. It seems that their definition is in another commit.